client/data/talks.json

```diff
@@ -449,6 +449,195 @@
       "video_url": "https://www.youtube.com/watch?v=MBndZddVQdw",
       "video_upload_date": "2019-06-03",
       "conferences": [1]
+    },
+    "50": {
+      "speakers": [63],
+      "main_title": "Redefining Web Performance",
+      "alternative_titles": [],
+      "categories": [5],
+      "video_url": "https://www.youtube.com/watch?v=JJGxNjrIYVc",
+      "video_upload_date": "2018-06-11",
+      "conferences": [2]
+    },
+    "51": {
+      "speakers": [64],
+      "main_title": "Hunting the Unicorn",
+      "alternative_titles": [],
+      "categories": [2, 5, 6],
+      "video_url": "https://www.youtube.com/watch?v=-PqL4qhvnE8",
+      "video_upload_date": "2018-06-11",
+      "conferences": [2]
+    },
+    "52": {
+      "speakers": [65],
+      "main_title": "Any Site can be a Progressive Web App",
+      "alternative_titles": [],
+      "categories": [1, 8],
+      "video_url": "https://www.youtube.com/watch?v=GXv4W94qsQ0",
+      "video_upload_date": "2018-06-11",
+      "conferences": [2]
+    },
+    "53": {
+      "speakers": [66],
+      "main_title": "It’s My (Third) Party and I'll Cry If I Want To",
+      "alternative_titles": [],
+      "categories": [5],
+      "video_url": "https://www.youtube.com/watch?v=rzMID7ES6q4",
+      "video_upload_date": "2018-06-11",
+      "conferences": [2]
+    },
+    "54": {
+      "speakers": [67],
+      "main_title": "A Rube Goldberg Machine and New Web Technology",
+      "alternative_titles": [],
+      "categories": [0, 1, 2],
+      "video_url": "https://www.youtube.com/watch?v=DCRN1cG7sOA",
+      "video_upload_date": "2018-06-11",
+      "conferences": [2]
+    },
+    "55": {
```

Comment:
So Henri had this talk the next year (2019) as well which is already in the database.
However, the content (and the presentation) of the talk is not the same so I think it's ok to have both.

BTW it's nice that the `conferences` is an array, but I think the `video_url` should be an array as well, so if in the future we will have the same talk recorded in multiple conferences then we could show all the videos. You know, maybe the video/audio is better, or the talk is longer/more detailed etc.

Reply:
Maybe (for now) it is best to just add the year to Henri's talk title for consecutive years. 

I think that's a good idea. My original idea is that we could just have the best version of the talk displayed here, but I can see how having multiple versions could be beneficial. However I'm not sure how this would be displayed in the UI without cluttering the talk card.

I added a new issue to discuss this here https://github.com/EddyVinck/ConfTalks/issues/9